web/html/forms-iv/formenctype.md

```diff
@@ -2,20 +2,26 @@
 author: Stefan-Stojanovic
 type: normal
 category: must-know
-links: null
+links:
+  - >-
+    [MDN enctype and formenctype](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form#attributes_for_form_submission){documentation}
 ---
 
-# The `formenctype` Attribute
+# enctype & formenctype Attributes
 
 
 ---
 
 ## Content
 
 The HTML `formenctype` input form attribute is used to specify how form-data should be encoded before it's sent to the server.
-It is only used for buttons and images with `type="submit"`.
+It is only used for buttons with `type="submit"` and `type="image"`.
 
-If the attribute is not set, the default value is set to `application/x-www-form-urlencoded`.
+An equivalent attribute is called `enctype`. It has the same functionality, except it is used on the form instead of the two mentioned button types.
+
+Also, the `formenctype` attribute overwrites the functionalities of `enctype` if both are present within the same form.
```

Comment:
```suggestion
When using the `formenctype` attribute (on the `form` element), its functionality is overwritten by the `enctype` attribute (on the `input` element) **if both are present within the same form**.
```

Reply:
But you can't use the `formenctype` on the form element, only on the buttons

Reply:
You can use it on the elements(buttons) inside the form, but on the form element itself

Reply:
Formenctype overwrites enctype. Same goes for formnovalidate > novalidate.

Reply:
Applied after name reverse